packages/api-specification/src/hiddenWindow.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
```

Comment:
Feels a little odd to me that this is in the api-specification and not in the demo packages - I guess the benefit having it here is that it doesn't need to be duplicated?